plugin-server/src/utils/db/db.ts

```diff
@@ -1396,47 +1387,14 @@ export class DB {
             plugin_id: pluginConfig.plugin_id.toString(),
         })
 
-        if (this.KAFKA_ENABLED) {
-            try {
-                await this.kafkaProducer.queueMessage({
-                    topic: KAFKA_PLUGIN_LOG_ENTRIES,
-                    messages: [{ key: parsedEntry.id, value: Buffer.from(JSON.stringify(parsedEntry)) }],
-                })
-            } catch (e) {
-                captureException(e)
-                console.error(parsedEntry)
-                console.error(e)
-            }
-        } else {
-            await this.postgresLogsWrapper.addLog(parsedEntry)
-        }
-    }
-
-    public async batchInsertPostgresLogs(entries: ParsedLogEntry[]): Promise<void> {
-        const LOG_ENTRY_COLUMN_COUNT = 9
-
-        const rowStrings: string[] = []
-        const values: any[] = []
-
-        for (let rowIndex = 0; rowIndex < entries.length; ++rowIndex) {
-            const { id, team_id, plugin_id, plugin_config_id, timestamp, source, type, message, instance_id } =
-                entries[rowIndex]
-
-            rowStrings.push(generatePostgresValuesString(LOG_ENTRY_COLUMN_COUNT, rowIndex))
-
-            values.push(id, team_id, plugin_id, plugin_config_id, timestamp, source, type, message, instance_id)
-        }
         try {
-            await this.postgresQuery(
-                `INSERT INTO posthog_pluginlogentry
-                (id, team_id, plugin_id, plugin_config_id, timestamp, source, type, message, instance_id)
-                VALUES
-                ${rowStrings.join(', ')}`,
-                values,
-                'insertPluginLogEntries'
-            )
+            await this.kafkaProducer.queueMessage({
+                topic: KAFKA_PLUGIN_LOG_ENTRIES,
+                messages: [{ key: parsedEntry.id, value: Buffer.from(JSON.stringify(parsedEntry)) }],
+            })
         } catch (e) {
             captureException(e)
+            console.error(parsedEntry)
```

Comment:
debug logging

Reply:
This is actually the diff being a bit wonky with indentation, since this is exactly the way it was in the `KAFKA_ENABLED` block already, but makes sense to remove.